### PR TITLE
Create INS1005Maaqud.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Maaquddi as institution/repository

### CODE REVIEW

File: new/INS1005Maaqud.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                            <placeName xml:id="n1" xml:lang="gez">መዓቁዲ፡ ዮሐንስ፡ ወልደ፡ ነጐድጓድ፡</placeName>
+                            <placeName corresp="#n1" xml:lang="en">Maʿāquddi Yoḥannǝs Walda Nagʷadgʷād</placeName>
+                            <placeName xml:id="n2" xml:lang="gez">ዮሐንስ፡ መዓቁዲ፡</placeName>
+                            <placeName corresp="#n2" xml:lang="en">Yoḥannǝs Maʿāquddi</placeName>
+                    <placeName xml:id="n3" xml:lang="gez">ዮሐንስ፡ መዓቁዲ፡ ደብረ፡ ከርቤ፡ </placeName>
+                    <placeName corresp="#n3" xml:lang="en">Yoḥannǝs Maʿāquddi Dabra Karbe </placeName>
+                    <placeName xml:id="n4" xml:lang="gez">ደብረ፡ ከርቤ፡ ዮሐንስ፡ መዓቁዲ፡</placeName>
+                    <placeName corresp="#n4" xml:lang="en">Yoḥannǝs Maʿāquddi Dabra Karbe</placeName>
+                            <placeName xml:id="n5" xml:lang="gez">ዮሐንስ፡ ዓዲ፡ አይኑ፡</placeName>               
+                            <placeName corresp="#n5" xml:lang="en">Yoḥannǝs ʿĀddi ʿĀynu</placeName>
```
Comment (thea-m) [inline new/INS1005Maaqud.xml:57]: ዓ or ʾA

File: new/INS1005Maaqud.xml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                                    <persName ref="PRS5695John" type="tabot">Yoḥannǝs Walda Nagʷadgʷād</persName>, 
+                                    <persName ref="PRS6819Mary" type="tabot">Mary</persName>
+                                    <persName ref="PRS10776Kidan" type="tabot">the Covenant of Mercy</persName>
+                                    <note>The third <foreign xml:lang="gez">tabot</foreign>, dedicated to <persName ref="PRS10776Kidan"/>, is hosted by a new church 
+                                    located in the vicinity.</note>                                    
+                                </ab>
+                                
+                                <ab type="description">Six manuscripts were digitized and 26 others were recorded by D. Nosnitsin on <date>15 November 2023</date>. 
+                                    Some more manuscripts remained unrecorded (among them are fragments of liturgical manuscripts, possibly of the 17th century, preserved inside the
+                                    rock-hewn church). Two wooden crosses, an old wooden <foreign xml:lang="gez">manbara tabot</foreign>
```
Comment (thea-m) [inline new/INS1005Maaqud.xml:84]: tābot

File: new/INS1005Maaqud.xml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                            </location>
+                            <note>
+                                <date type="foundation" notBefore="1400" notAfter="1499">Possibly 15th century</date>
+                                <ab type="history">Dabr. Rock-hewn church well-known for its frescos depicting <persName ref="PRS4015Fasilada">King Fasiladas</persName>.
+                                </ab>
+                                <ab type="tabot">
+                                    <measure quantity="2"></measure>                                   
+                                    <persName ref="PRS5695John" type="tabot">Yoḥannǝs Walda Nagʷadgʷād</persName>, 
+                                    <persName ref="PRS6819Mary" type="tabot">Mary</persName>
+                                    <persName ref="PRS10776Kidan" type="tabot">the Covenant of Mercy</persName>
```
Comment (eu-genia) [inline new/INS1005Maaqud.xml:77]: `PRS10776Kidan `does not exist, we do not have PRS records for non-persons. You should link to the keyword `CovenantMercy`
`<ref type="authFile" corresp="CovenantMercy">Kidāna Mǝḥrat<ref>`

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, very nice

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/INS1005Maaqud.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/INS1005Maaqud.xml:84]: :)

File: new/INS1005Maaqud.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/INS1005Maaqud.xml:77]: ok

Comment (eu-genia) [reply new/INS1005Maaqud.xml:77]: not corrected yet?

Comment (DenisNosnitsin1970) [reply new/INS1005Maaqud.xml]: sorry, one update was forgotten.

File: new/INS1005Maaqud.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/INS1005Maaqud.xml:57]: ok

**Review by eu-genia — APPROVED** [review]

**Pull request merged**